### DISCUSSION TITLE
Incorporação das informações dos arquivos CNV e DEF

### DISCUSSION THREAD

**Discussion by momenezes** [rfsaldanha/microdatasus#85] [Ideas]
Tem uma biblioteca em Python que faz a incorporação das definições CNV e DEF quando se lê os arquivos DBC do SIHSUS (https://github.com/gcms/tabsus). 
Penso que seria uma funcionalidade interessante, já que permitiria uma análise muito mais informativa dos dados.
Alguém já fez algo parecido em R?

**Comment by rfsaldanha** [comment]
Olá! Não conhecia a biblioteca, obrigado por avisar. Realmente, poder fazer um parsing das definições dos CNF e DEF seria muito produtivo... Por outro lado, a biblioteca está sem atualizações há dois anos, o que pode ser um ponto de fragilidade ao incorporá-la ao microdatasus. Ou será que não foram necessárias atualizações da biblioteca?

**Reply by mariomack** [reply]
Oi Rafael, sim, a biblioteca está sem atualizações; não sei se "abandonada" ou só não foram necessárias. 
Eu vou tentar usá-la com o `reticulate` e ver o que dá. Vc acha que essa seria a melhor estratégia (usar via `reticulate`) ou reescrevê-la como uma _library_ do **R**? 
Abs

**Reply by momenezes** [reply]
oops ... minha resposta acabou indo por minha outra conta, só percebi depois ...

**Reply by rfsaldanha** [reply]
Acho que eu usaria o `reticulate` no começo. Depois de algum uso, podemos pensar se é necessário ou interessante re-implementar em R. Enfim, testes e mais testes....
